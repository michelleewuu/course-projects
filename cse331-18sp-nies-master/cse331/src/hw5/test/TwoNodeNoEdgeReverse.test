# Same as TwoNodeNoEdge.test, but reverse the 
# order of node adding to test if it prints 
# out in alphabetical order

# Create a graph
CreateGraph two_nodes_graph_reverse

# Add two nodes in reverse alphabetical order
AddNode two_nodes_graph_reverse b
AddNode two_nodes_graph_reverse a

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_graph_reverse
ListChildren two_nodes_graph_reverse a
ListChildren two_nodes_graph_reverse b