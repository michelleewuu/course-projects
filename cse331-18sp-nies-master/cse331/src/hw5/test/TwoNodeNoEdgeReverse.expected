# Same as TwoNodeNoEdge.test, but reverse the 
# order of node adding to test if it prints 
# out in alphabetical order

# Create a graph
created graph two_nodes_graph_reverse

# Add two nodes in reverse alphabetical order
added node b to two_nodes_graph_reverse
added node a to two_nodes_graph_reverse

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_graph_reverse contains: a b
the children of a in two_nodes_graph_reverse are:
the children of b in two_nodes_graph_reverse are: